Taarafo.Core/Services/Foundations/PostReports/PostReportService.Validations.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ---------------------------------------------------------------
```

Comment:
You should create and write this code in the PostImpressionService.Exceptions.cs
not in the PostImpressionService.Validations.cs file